tests/config/checker/checkers/test_gLobal_config_checker.py

```diff
@@ -19,6 +19,42 @@
 
 
```

Comment:
if we decide to go with my proposal above, we will need to inject the value here for testing purposes (or in the conftest.py file or within each unit tests in this file where appropriate)
```suggestion
_GlobalConfigChecker. _ACCEPTED_REPOSITORY_TYPES.extend(["filesystem", "sql"])

```

Reply:
I think we can just use ["mock_repo"] to keep the test here simple. Then we will test for actual repository types in taipy-core and else where.